### ISSUE TITLE
React native environment setup issue in Apple M2 Pro

### ISSUE THREAD

**Issue by kavi2017** [react-native-community/.github#79]
I am new to React native development in Mac M2 pro. I started React native environment set up. But all the time I face this issue 
"✖ Installing Ruby Gems
error Looks like your iOS environment is not properly set"
Tried to resolve with cocapods and gemfile fixes.
I am also tried to resolve as per online guidelines  still not getting solution. Kindly help to resolve issue to start my react native journey. Thanks in advance

**Comment by kelset** [comment]
👋 there - this repository is archived and shouldn't be used. Yours seems more a question for [StackOverflow instead](https://stackoverflow.com/questions/tagged/react-native).

**Comment by kavi2017** [comment]
@kelset Yes I asked question only. I am looking support from github users
